src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx

```diff
@@ -20,8 +23,9 @@ const MaterialButtonsFindOnShelf: FC<MaterialButtonsFindOnShelfProps> = ({
     recordid: faustIds
   });
 
+  const dispatch = useDispatch();
   const onClick = () => {
-    // we open the find on shelf modal here
+    dispatch(openModal({ modalId: findOnShelfModalId(faustIds[0]) }));
```

Comment:
https://github.com/danskernesdigitalebibliotek/dpl-react/blob/c16465c5cccd4b2f116f49b392e11541fcaa3348/src/core/utils/modal.tsx#L88

:)

Reply:
That's right! :)